content/tutorials/Team13Tutorial.md

```diff
@@ -0,0 +1,244 @@
+
+### Introduction
+
+In this section, we will be discussing the voltage divider and circuit setup. We will not be writing any code, so a computer is not yet needed.
+
+### Objective
+
+- Understand how to pick values for a voltage divider.
+- Understand how to properly connect the battery, resistors, and ESP32.
+
+### Background Information
+
+Give a brief explanation of the technical skills learned/needed
+in this challenge. There is no need to go into detail as a
+separation document should be prepared to explain more in depth
+about the technical skills
+
+If you've taken a look at the battery we gave you, you might notice that it says 3.7V on it. The maximum voltage input for the ESP32 is 3.3V, so we need to find a way to decrease the voltage from the battery, we can do this with a <b>voltage divider</b>.
+
+A voltage divider is a passive linear circuit made up of two resistors, although sometimes more are used to get a specific resistance value. 
+![Voltage Divider Schematic and Equation](Team13Photos/voltagedivider.png)
+
+We have given you two 47kΩ resistors, so the output voltage will be 3.7V * 47kΩ/(47kΩ + 47kΩ) = 1.85V. 
+
+Notice that our voltage divider halves the input voltage. To combat this division, we will need to remember to multiply our measured voltage by 2 (the reciprocal).
+
+### Components
+
+- Breadboard
+- Battery and Battery Holder
+- Resistors
+- ESP32
+- Jumper Cables
+
+### Instructional
+
+Assemble the circuit as shown in the below image. 
+![Circuit Fritz](Team13Photos/Fritz.png)
+Make sure to attach the cathode of the battery to the voltage divider *AND* to the ground pin of the ESP32!
+
+## Part 02: Writing the Code
+
+### Introduction
+
+In this section, we will be discussing the code used to measure the battery's percentage. 
+
+### Objective
+
+- Understand the Arduino interface.
+- Understand how to assign pin numbers in Arduino.
+- Understand how to implement functions in Arduino.
+
+### Background Information
+
+This section focuses on coding in the Arduino IDE. The language is very similar to C or C++. If you have any background knowledge in those languages, you will recognize the code and its formatting. If you don't have experience in C/C++, don't worry, we will be breaking down the code into very manageable chunks. 
+*Hint: If you don't want to rewrite all the code, use the copy button in the upper right corner of the code blocks!*
+
+### Components
+
+- ESP32 (with the circuit you just built attached)
+- USB-C to USB-C Connector cable for ESP32
+- Your Computer
+
+### Instructional
+
+Another goal of the project is to integrate both hardware and software to create a simple battery indicator system using the ESP32 microcontroller.
+Let's take a dive into the indvidual blocks of code. 
+
+```
+{
+  const int batteryPin = 1;
+  const float referenceVoltage = 3.3;
+  const int resolution = 3950;
+  const float voltageDividerRatio = 2.0;
+  const float maxBatt = 4.2;
+  const float minBatt = 3;
+}
+```
```

Comment:
this looks like arduino code (which is C). you should mark the code blocks as C so syntax highlighting is enabled.